demos/cuj1-gke-config.md

```
@@ -0,0 +1,314 @@
+# AICR - CUJ 1 — GKE (config-driven + evidence)
+
+Same flow as [`cuj1-gke.md`](./cuj1-gke.md), but driven by a single `AICRConfig`
+file via `--config` for `recipe`, `bundle`, and `validate`, and finished with
+`--emit-attestation` + `aicr evidence verify` (see [`evidence.md`](./evidence.md)
+for the standalone evidence walkthrough). Reproducible inputs in, signed
+recipe-evidence bundle out.
+
+> Note: This demo duplicates a lot of the validation demo from demos/evidence.md. 
+>       The will be to generalize the cluster validation and evidence collection 
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)